mods/cnc/weapons/missiles.yaml

```diff
@@ -377,16 +377,12 @@ Patriot:
 		RateOfTurn: 20
 		Speed: 341
 		RangeLimit: 30
-		Angle: 88
```

Comment:
Wouldn't the removal of this trigger the circling-missile behaviour on directly above targets?